canonical/vanilla-framework
# Percy tokens prep

## Done

Makes Percy changes necessary for testing Style Dictionary PRs.

Between time this is merged and Style Dictionary is merged, the `yarn build-tokens` command will fail but its failure will be ignored. This is so that Percy job runs by Style Dictionary PRs use the `package.json` from main (including style dictionary dependency, scripts, & workflows).

Style dictionary PR can then actually add token definitions & config and call the tokens build script introduced by this PR in testing. Style dictionary PR should also remove the `|| true` in `build-tokens` NPM script so that future tokens build failures are not ignored

## QA

- Verify that the [PR prepare](https://github.com/jmuzina/vanilla-framework/actions/runs/10319841061) and [snapshot](https://github.com/jmuzina/vanilla-framework/actions/runs/10319844817) workflows run successfully for [PR](https://github.com/jmuzina/vanilla-framework/pull/32) when this PR's code has been pushed to [main](https://github.com/jmuzina/vanilla-framework) already


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)
